cmd/root.go

```diff
@@ -52,7 +52,17 @@ var (
 
 func init() {
 	versionString = strings.TrimSpace(versionString)
-	userAgent = "cloud-sql-proxy/" + versionString
+	userAgent = userAgentString()
+}
+
+func userAgentString() string {
+	ua := "cloud-sql-proxy/" + versionString
+	runtime, ok := os.LookupEnv("CLOUD_SQL_PROXY_RUNTIME")
```

Comment:
@kurtisvg pointed out that some runtimes (e.g., Cloud run where they'll use this programatically) might not be able to set an environment variable.

Let's make this a CLI flag proper (which will also support an environment variable).

I'm in favor of a single flag rather than a repeatable flag.